apps/test/unit/lib/kits/maker/portScanningTest.js

```diff
@@ -9,38 +9,28 @@ import {
   OSX_DEFAULT_PORTS,
   OTHER_BAD_SERIALPORTS,
 } from './sampleSerialPorts';
-import ChromeSerialPort from 'chrome-serialport'; // Actually StubChromeSerialPort
 import {ConnectionFailedError} from '@cdo/apps/lib/kits/maker/MakerError';
 import {
   findPortWithViableDevice,
   getPreferredPort,
 } from '@cdo/apps/lib/kits/maker/portScanning';
-import sinon from 'sinon';
 
 describe('maker/portScanning.js', function () {
-  let userAgentSpy;
   describe(`findPortWithViableDevice()`, () => {
-    beforeEach(() => {
-      // 'CrOS' represents ChromeOS
-      userAgentSpy = sinon.stub(navigator, 'userAgent').value('CrOS');
-    });
-
-    // Testing against StubChromeSerialPort.js
-    afterEach(() => {
-      ChromeSerialPort.stub.reset();
-      userAgentSpy.restore();
-    });
-
     it('resolves with a port if a viable device is found', () => {
-      ChromeSerialPort.stub.setDeviceList(CIRCUIT_PLAYGROUND_PORTS);
+      window.SerialPort = {
```

Comment:
Any issues with setting this on `window` directly? Do we need to clean this up in an `afterEach` block?

Reply:
This unit test will be removed in an upcoming PR (draft [here](https://github.com/code-dot-org/code-dot-org/pull/52435))because the function `findPortWithViableDevice` will be removed. This is a temp measure so that tests can pass for now.